cherab/solps/plotting.py

```diff
@@ -0,0 +1,100 @@
+import matplotlib.pyplot as plt
```

Comment:
I would suggest renaming the file to plot.py to match the convention from core package.